build.sbt

```diff
@@ -167,15 +167,58 @@ lazy val D_Shared = new {
 }
 
 lazy val D_SharedTest = new {
-  lazy val dependencies: Seq[ModuleID] =
+  // https://github.com/sbt/sbt-license-report/issues/87
+  // https://stackoverflow.com/questions/48771768/sbt-error-importing-resteasy-client
+  //
+  // 'sbt-license' plugin is using ivy to resolve dependencies where other tasks are using coursier.
+  // 'org.jboss.resteasy:resteasy-*' which is the transitive dependencies of 'keycloak-admin-client'
+  // has this issue where 'relativePath' is used in the 'parent' section.
+  // - https://github.com/resteasy/resteasy/blob/6.2.4.Final/resteasy-client-api/pom.xml#L9
+  // - https://www.scala-sbt.org/1.x/docs/Library-Management.html#Known+limitations
+  //
+  // This workaround provides those dependencies explicitly, but it will be a nightmare to maintain.
+  // for version reference: https://github.com/resteasy/resteasy/blob/6.2.4.Final/resteasy-dependencies-bom/pom.xml
+  // FIXME: solve this with a long-term solution
+  lazy val keycloakAdminExplicitDependencies: Seq[ModuleID] =
```

Comment:
Should we also fix a version of keycloak test containers for Scala steward to never update it automatically? Or I believe, we'll have a situation when this list will be outdated and we'll have this issue once again.

Reply:
I've turned it off for `keycloak-testcontainer` and `keycloak`. They should go together and use same the keycloak version.